packages/sandbox/src/compilerOptions.ts

```diff
@@ -85,7 +85,7 @@ export const getCompilerOptionsFromParams = (
       let toSet = undefined
       if (val === "true" && playgroundDefaults[key] !== true) {
         toSet = true
-      } else if (val === "false" && playgroundDefaults[key] !== false) {
+      } else if (val === "false" && (playgroundDefaults[key] as any) !== false) { // TODO(jakebailey): remove as any, check undefined above
```

Comment:
I think this may have been hiding a bug, but I don't want to change this PR to modify the behavior in this PR.